Sources/EduIDExpansion/MainCoordinator.swift

```diff
@@ -19,12 +19,16 @@ class MainCoordinator: CoordinatorType {
     }
     
     func start() {
-        let onboardinCoordinator =  OnboardingCoordinator(navigationController: self.navigationController)
+        let onboardinCoordinator = OnboardingCoordinator(navigationController: self.navigationController)
         children.append(onboardinCoordinator)
         onboardinCoordinator.start()
     }
     
-    func showNextScreen(currentType: ScreenType) {
+    func goBack() {
+        print("does nothing")
```

Comment:
I think this method shouldn't exist on MainCoordinator.

Reply:
I moved those methods to onboardingCoordinator